tests/base.conf.js

```diff
@@ -41,6 +43,16 @@ class BaseConfig {
         browser.driver.wait(startApi(), 135 * 1000, 'API took too long to start up');
         browser.driver.sleep(10000); // wait for startup to complete
 
+        afterEach(() => {
+          browser.manage().logs().get('browser').then(logs => {
+            const formatted = logs
+              .filter(log => log.level.value_ > 900)
```

Comment:
Maybe move 900 into a variable or add a comment indicating the significance of this value